monai/bundle/properties.py

```diff
@@ -48,6 +48,11 @@
         BundleProperty.REQUIRED: True,
         BundlePropertyConfig.ID: f"train{ID_SEP_KEY}trainer",
     },
+    "network_def": {
```

Comment:
May I know why add this property? We may need to update the CI test of model-zoo for it, CC @yiheng-wang-nv .

Thanks.

Reply:
Hi @Nic-Ma, I added this property, so that in the `load` API, it can directly use the network in the bundle.
https://github.com/Project-MONAI/MONAI/blob/5da4ea6c8e05f123c5d53913037950a05daa8985/monai/bundle/scripts.py#L505

Thanks!

Reply:
OK, I see.